pkg/minikube/cluster/cluster.go

```diff
@@ -223,11 +224,34 @@ var assets = []fileToCopy{
 	},
 }
 
-func UpdateCluster(d drivers.Driver) error {
+func UpdateCluster(h sshAble, d drivers.Driver, config KubernetesConfig) error {
 	client, err := sshutil.NewSSHClient(d)
 	if err != nil {
 		return err
 	}
+	if localkubeURLWasSpecified(config) {
+		resp, err := http.Get(util.GetLocalkubeDownloadURL(config.KubernetesVersion,
+			constants.LocalkubeLinuxFilename))
+		if err != nil {
+			return err
+		}
+		if err := sshutil.Transfer(resp.Body, int(resp.ContentLength), "/usr/local/bin",
```

Comment:
How do we recover from interrupted downloads? Just restart? Does this get downloaded on every stop/start or is it cached?